src/main/io/serial_msp.c

```diff
@@ -1878,13 +1878,10 @@ static bool processInCommand(void)
         break;
         
     case MSP_SET_NAME:
-        memset(masterConfig.name, 0, MAX_NAME_LENGTH+1);
+        memset(masterConfig.name, 0, ARRAYLEN(masterConfig.name)); 
```

Comment:
Thanks for these improvements. I searched for ARRAYLEN (as per your feedback yesterday) in some C documentations but never found it but realize now that it was a defined macro :)